src/css/toolbar.css

```diff
@@ -4,20 +4,21 @@
   /* box-shadow: 0 1px 0 var(--toolbar-border-color); */
   display: flex;
   font-size: calc(14.5 / var(--rem-base) * 1rem);
-  height: var(--toolbar-height);
```

Comment:
I think using `var(--toolbar-height)` here is important because some of the page margins rely on knowing how tall the toolbar is to align things properly. A better approach is to change the --toolbar-height var in vars.css

Reply:
Already fixed